### ISSUE TITLE
Mariadb container for 1.14.0 and 1.14.1 versions fail when updating apt packages

### ISSUE THREAD

**Issue by rthideaway** [drud/ddev#2362]
**Describe the bug**
Mariadb container fails with:
```
Successfully built d44a0769fe86
Successfully tagged drud/ddev-dbserver-mariadb-10.2:v1.14.1-jobiqo8-built
Step 1/11 : ARG BASE_IMAGE
Step 2/11 : FROM $BASE_IMAGE
 ---> 46abc0aeb4b9
Step 3/11 : RUN apt update
 ---> Running in a7501bf8dee8

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Get:1 http://security.debian.org buster/updates InRelease [65.4 kB]
Get:2 http://nginx.org/packages/debian buster InRelease [2848 B]
Get:3 http://deb.debian.org/debian buster InRelease [121 kB]
Get:4 https://dl.yarnpkg.com/debian stable InRelease [17.1 kB]
Get:5 https://packages.sury.org/php buster InRelease [6759 B]
Get:6 http://nginx.org/packages/debian buster/nginx amd64 Packages [9701 B]
Get:7 https://dl.yarnpkg.com/debian stable/main amd64 Packages [10.0 kB]
Get:8 https://deb.nodesource.com/node_12.x buster InRelease [4584 B]
Get:9 http://security.debian.org buster/updates/main amd64 Packages [260 kB]
Get:10 https://packages.sury.org/php buster/main amd64 Packages [132 kB]
Get:11 https://dl.yarnpkg.com/debian stable/main all Packages [10.0 kB]
Get:12 http://packages.blackfire.io/debian any InRelease [23.2 kB]
Get:13 https://deb.nodesource.com/node_12.x buster/main amd64 Packages [763 B]
Get:14 http://deb.debian.org/debian buster/main amd64 Packages [10.7 MB]
Err:10 https://packages.sury.org/php buster/main amd64 Packages
  Hash Sum mismatch
  Hashes of expected file:
   - Filesize:132039 [weak]
   - SHA256:3476471816bac3aa98bf4734dafc107c35e192ed460b32a84112402b5221cb7a
   - SHA1:4e71be38cffc2c22662c7e7271ea2b4caf7930b0 [weak]
   - MD5Sum:3a7f245ffe0a311a8c77a0af4182eef5 [weak]
  Hashes of received file:
   - SHA256:1501bcecbe2592caa11cd80048cd0d8f9bdda0baf4d7fb1e5b165182b27ab692
   - SHA1:5ebdce3a2548f1e2d7db89323a237d5d3618f636 [weak]
   - MD5Sum:b7302821f0c0fe5ec8b5276a136fc143 [weak]
   - Filesize:132039 [weak]
  Last modification reported: Tue, 07 Jul 2020 06:58:38 +0000
  Release file created at: Fri, 03 Jul 2020 12:15:19 +0000
Get:15 http://packages.blackfire.io/debian any/main amd64 Packages [1101 B]
Fetched 11.4 MB in 1min 37s (118 kB/s)
Reading package lists...
E: Failed to fetch https://packages.sury.org/php/dists/buster/main/binary-amd64/Packages.gz  Hash Sum mismatch
   Hashes of expected file:
    - Filesize:132039 [weak]
    - SHA256:3476471816bac3aa98bf4734dafc107c35e192ed460b32a84112402b5221cb7a
    - SHA1:4e71be38cffc2c22662c7e7271ea2b4caf7930b0 [weak]
    - MD5Sum:3a7f245ffe0a311a8c77a0af4182eef5 [weak]
   Hashes of received file:
    - SHA256:1501bcecbe2592caa11cd80048cd0d8f9bdda0baf4d7fb1e5b165182b27ab692
    - SHA1:5ebdce3a2548f1e2d7db89323a237d5d3618f636 [weak]
    - MD5Sum:b7302821f0c0fe5ec8b5276a136fc143 [weak]
    - Filesize:132039 [weak]
   Last modification reported: Tue, 07 Jul 2020 06:58:38 +0000
   Release file created at: Fri, 03 Jul 2020 12:15:19 +0000
E: Some index files failed to download. They have been ignored, or old ones used instead.
```

**To Reproduce**
Use version 1.14.0 or 1.14.1 and run `ddev start` on your project

**Expected behavior**
No error, mariadb works just fine

**Screenshots**
![mariadb-container-fails-on-apt-update](https://user-images.githubusercontent.com/5792505/86747627-29cb9500-c03c-11ea-97d7-651f1a7a3df3.png)

**Version and configuration information (please complete the following information):**
 - Host computer OS and Version: Ubuntu 20.04
 - Docker version: 19.03.11
 - ddev version: 1.14.1

**Additional context**
This makes all our builds failing, as we use ddev on CI to test the app :/

**Comment by rthideaway** [comment]
actually works with 1.14.2, so I guess this can be closed, we updated version on CI and now we are fine

**Comment by rfay** [comment]
I'd be interested to know what extra packages you were adding on the db container, thanks. I see people add lots of webimage_extra_packages, but haven't seen people doing it on the db container.

**Comment by rthideaway** [comment]
actually we added no extra packages for db container.

**Comment by rfay** [comment]
Ah, I see, you titled it "Mariadb", but that was just the first line (mariadb built successfully). The problem was with building the web image.

However, the actual failure you had with the sury packages seems pretty strange, I'm surprised at it. But glad it's working for you.

**Comment by rthideaway** [comment]
Ah, my bad, I'm sorry! Regarding web container we were using those extra packages:
```
webimage_extra_packages: [optipng, libjpeg-progs]
```
Anyway, this problem seems to be resolved now even for 1.14.0 and 1.14.1 versions, not sure why it was happening yesterday
